docs/_docs/06-dg-building-from-source.md

```diff
@@ -2,7 +2,39 @@
 title: "Building from Source"
 permalink: /docs/dg-building-from-source/
 excerpt: "How to configure Hyperspace for your needs."
-last_modified_at: 2020-06-20
+last_modified_at: 2020-06-23
 toc: true
 ---
 
+## Building from Source
+
+Hyperspace is built with [sbt](https://www.scala-sbt.org/). Run the following commands from the project root directory.
+
+### On *nix OS
+
```

Comment:
Can you remove this section from the main README.md?

Reply:
removed. Thanks @imback82